README.md

```diff
@@ -25,3 +25,27 @@ and run it:
    py35: commands succeeded
    congratulations :)
 ```
+
+## Deploying
+
+**devpi**  
+*Dev libraries : http://pypi.skimlinks.net/skim/dev*  
+*Prod libraries : http://pypi.skimlinks.net/skim/live*  
```

Comment:
is this repo still an open source public repo? not sure we want to put those info if it is...